paper/paper.md

```diff
@@ -0,0 +1,170 @@
+  - affiliation: 12
+    name: dominikwelke
+    orcid: xxxx-xxxx-xxxx-xxxx
+  - affiliation: 13
+    name: Clemens Brunner
+    orcid: 0000-0002-6030-2233
+  - affiliation: 14
+    name: Alexander P Rockhill
+    orcid: 0000-0003-3868-7453
+  - affiliation: 15
+    name: Alexandre Gramfort
+    orcid: 0000-0001-9791-4404
+affiliations:
+- index: 1
+  name: Center for Adaptive Rationality, Max Planck Institute for Human Development, Berlin, Germany
+- index: 2
+  name: Athinoula A. Martinos Center for Biomedical Imaging, Massachusetts General Hospital, Charlestown, MA, USA
+- index: 3
+  name:
+- index: 4
+  name: Mozilla
+- index: 5
+  name: Department of Neuroscience and Biomedical Engineering, Aalto University, Espoo, Finland
+- index: 6
+  name: Human Cortical Physiology and Neurorehabilitation Section, NINDS, NIH, Bethesda, Maryland 20892
+- index: 7
+  name: UC Berkeley, Project Jupyter
+- index: 8
+  name:
+- index: 9
+  name: Department of Biomedical and Clinical Sciences 'L. Sacco', University of Milan, Milan, Italy
+- index: 10
+  name:
+- index: 11
+  name:
+- index: 12
+  name:
+- index: 13
+  name: Institute of Psychology, University of Graz, Austria
+- index: 14
+  name: University of Oregon, Eugene OR, USA
+- index: 15
+  name: Université Paris-Saclay, Inria, CEA, Palaiseau, France
+date: 28 October 2019
+bibliography: paper.bib
+---
+
+# Summary
+
+With the emergence of the Brain Imaging Data Structure
+(``BIDS``; [@Gorgolewski2016]) in 2016, the scientific community received a
+standard to organize and share data in the broad domain of neuroscience.
+
+Originally limited to MRI data types, BIDS is continuously being extended and
+by now also supports other neuroimaging modalities such as MEG [@Niso2018],
+EEG [@Pernet2019], and iEEG [@Holdgraf2019].
+
+BIDS prescribes how complex experiment data should be structured and which
+metadata should be encoded next to the raw data. This set of guidelines
+allows users to reap several benefits, such as:
+
+1. **sharing data within a lab and between labs:** Through reading the BIDS
+   specification, users are empowered to understand any BIDS dataset
+   without requiring specialist knowledge.
+1. **data validation:** Dedicated software can check whether the rules of
+   BIDS are being followed in a dataset, and alert users to missing or broken
+   data (see
+   [bids-validator](https://github.com/bids-standard/bids-validator)).
+1. **automatic data handling and analysis pipelines:** With a perfect
+   knowledge about which data to expect and where to search for it, many
+   common workflows in data handling and analysis can be simplified or
+   automated through specialized software.
+
+To make use of these benefits, datasets must first be converted to
+BIDS format. Furthermore, already existing data analysis software must be
+extended to be aware of a BIDS format and how to properly use it.
+
+This is where ``MNE-BIDS`` steps in: Harnessing BIDS and the data analysis
```

Comment:
... and the Python-based data analysis toolbox MNE-Python...

Reply:
-1 for 'Python-based data analysis toolbox', I find just 'data analysis software' more elegant. 'Python' is already in the 'MNE-Python' name.

Reply:
I left it with "data analysis software" for now, but will leave this comment open for others to tip the scales